### DISCUSSION TITLE
Question about pybamm.lithium_ion.get_initial_stoichiometries()

### DISCUSSION THREAD

**Discussion by Weisserlinks** [pybamm-team/PyBaMM#2038] [Q&A] [answered]
Dear Pybamm team,
Recently I've been trying to do aging-simulation of the Samsung 35E battery. In order to calculate the capacity, I chose to use pybamm.lithium_ion.get_initial_stoichiometries() function to get the stoichiometry both in SOC=0 and 1 and then did the calculation: `C_0 = (x_100 - x_0) * solution["Negative electrode capacity [A.h]"].data[0]`. For unknown reasons there was a small "jump" of my results on some batteries as the picture attached shows. I managed to figure out that it was due to the sudden change of the variable 'x_100' ---- stoichiometry of negative electrode at SOC=1. Would anyone happen to know the reason?
Many thanks in advance :)
![InkedCapacity curve_LI](https://user-images.githubusercontent.com/95657044/165362735-ac046fba-fca1-4443-aeb3-9ee3ede6a97d.jpg)
![Stoichiometries](https://user-images.githubusercontent.com/95657044/165362762-07c65bfd-e55f-4ba8-becc-8b09be320be8.jpg)

**Comment by valentinsulzer** [comment, marked as answer]
It looks like numerical error in the algebraic equation solve that is used to solve for x_100. Maybe a non-monotonic OCV? I'm planning on working on improving this solver and debugging tools around it

**Reply by Weisserlinks** [reply]
Thanks for your answer! I'm now avoiding this problem by simulating less cycles.
